website_sale_order_type/tests/test_website_sale_order_type.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.tests import HttpCase
+
+
+class TestFrontend(HttpCase):
+    def setUp(self):
+        super(TestFrontend, self).setUp()
+        self.sale_type_model = self.env["sale.order.type"]
+
+        self.partner = self.env.ref("base.partner_admin")
+        self.sale_type = self.create_sale_type()
+
+    def create_sale_type(self):
+        self.sequence = self.env["ir.sequence"].create(
+            {
+                "name": "Test Sales Order",
+                "code": "sale.order",
+                "prefix": "TSO",
+                "padding": 3,
+            }
+        )
+        self.journal = self.env["account.journal"].search(
+            [("type", "=", "sale")], limit=1
+        )
+        self.warehouse = self.env.ref("stock.stock_warehouse_shop0")
```

Comment:
As I see from the logs, this warehouse missmatches the one from the order company. Maybe that's avoiding the tour to finish correctly

Reply:
Thanks @chienandalu, I finally manage to get the test working.